scalalib/src/mill/scalalib/TestModule.scala

```diff
@@ -97,15 +97,24 @@ trait TestModule
   }
 
   /**
-   * How the test classes in this module will be split into multiple JVM processes
-   * and run in parallel during testing. Defaults to all of them running in one process
-   * sequentially, but can be overridden to split them into separate groups that run
-   * in parallel.
+   * How the test classes in this module will be split.
+   * Test classes from different groups are ensured to never
+   * run on the same JVM process, and therefore can be run in parallel.
+   * When used in combination with [[useTestStealingScheduler]],
+   * every JVM test running process will guarantee to never steal tests
+   * from different test groups.
    */
   def testForkGrouping: T[Seq[Seq[String]]] = Task {
     Seq(discoveredTestClasses())
   }
 
+  /**
+   * Whether to use the test stealing scheduler to run tests in multiple JVM processes.
+   * When used in combination with [[testForkGrouping]], every JVM test running process
+   * will guarantee to never steal tests from different test groups.
+   */
+  def useTestStealingScheduler: T[Boolean] = T(false)
```

Comment:
Let's call this `testEnableWorkStealing`? I'd like it to start with the `test*` prefix following the convention of the rest of the module's tasks